sklearn/preprocessing/data.py

```diff
@@ -1346,15 +1356,18 @@ def normalize(X, norm='l2', axis=1, copy=True, return_norm=False):
         X = X.T
 
     if sparse.issparse(X):
+        if return_norm and norm in ('l1', 'l2'):
+            raise NotImplementedError("'return_norm' isn't implemented for"
```

Comment:
Missing space after "for".

I would rephrase it slightly: "return_norm=True is not implemented for sparse matrices with norm 'l1' or 'l2'"